matrix/lambdas/daemons/driver.py

```diff
@@ -1,35 +1,90 @@
-import itertools
-import math
 import typing
+import os
 
 import requests
 
-from matrix.common.aws.lambda_handler import LambdaHandler, LambdaName
-from matrix.common.aws.cloudwatch_handler import CloudwatchHandler, MetricName
+from matrix.common.config import MatrixInfraConfig, MatrixRedshiftConfig
 from matrix.common.logging import Logging
-from matrix.common.request.request_cache import RequestCache
 from matrix.common.request.request_tracker import RequestTracker, Subtask
+from matrix.common.aws.dynamo_handler import DynamoHandler
+from matrix.common.aws.sqs_handler import SQSHandler
+from matrix.common.aws.s3_handler import S3Handler
 
 logger = Logging.get_logger(__name__)
 
+expression_query_template = """
+    UNLOAD ($$SELECT cell.cellkey, expression.featurekey, expression.exrpvalue
+    FROM expression
+    LEFT OUTER JOIN feature on (expression.featurekey = feature.featurekey)
+    INNER JOIN cell on (expression.cellkey = cell.cellkey)
+    INNER JOIN analysis on (cell.analysiskey = analysis.analysiskey)
+    WHERE feature.isgene
+    AND expression.exprtype = 'Count'
+    AND analysis.bundle_fqid IN {3}$$)
+    TO 's3://{0}/{1}/expression'
+    IAM_ROLE '{2}'
+    GZIP
+    ALLOWOVERWRITE
```

Comment:
I think we should remove `ALLOWOVERWRITE` from these. If the files already exist, something has gone wrong.

Reply:
Done